test/parallel/test-repl-history-navigation.js

```diff
@@ -68,6 +68,8 @@ const prev = process.features.inspector;
 
 let completions = 0;
 
+process.on('warning', common.mustNotCall());
```

Comment:
nit: Could you move this just after line 18? I feel it would be better to group all the `process` things together :)